src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/AutoSuggestBoxTests/BasicAutoSuggestBox.xaml.cs

```diff
@@ -29,11 +29,17 @@ public BasicAutoSuggestBox()
 			this.InitializeComponent();
 		}
 
+		private bool ShouldClear => ShouldClearTextBox.IsChecked.Value;
```

Comment:
```suggestion
		private bool ShouldClear => ShouldClearTextBox.IsChecked.GetValueOrDefault();
```

Even if it's just a sample, we should try to show good practices :)